framework/src/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -725,4 +732,28 @@ protected Boolean reduce(Boolean r1, Boolean r2) {
             }
         }
     }
+
+    private static List<String> getExpressionElementNames(Class<? extends Annotation> clazz) {
```

Comment:
Please add Javadoc.
That would probably also help me understand why this cannot call `getListOfExpressionElements`.